peregrine/resources/submission/graphql/node.py

```diff
@@ -795,6 +797,67 @@ def _make_inner_meta_type():
     return gql_object
 
 
+class NodeCounter:
+    """A workaround for performance issue in PXD-2336.
```

Comment:
let's not mention jira ticket number in any code. The code is open source but jira is not.
I think the below descriptions already described the issue well enough

Reply:
Okay, also removed from commit message. It is okay to [mention JIRA ticket number in PR title](https://gen3.org/resources/developer/dev-introduction/#github-usage)?

Reply:
Yeah, you can include it in the PR title if you want JIRA to link to the PR automatically